### ISSUE TITLE
Rendering issue with readthedocs

### ISSUE THREAD

**Issue by dmed256** [djungelorm/sphinx-tabs#1]
I happened to see this repo when I wanted to include tabs in readthedocs
Switching between tab contents worked!

However, there seems to be some css/rending issues

![tabs](https://cloud.githubusercontent.com/assets/1812355/22628699/827b3438-eb9e-11e6-9106-4ce894fecb79.png)

It looks like the html generated was right

![html](https://cloud.githubusercontent.com/assets/1812355/22628730/c6d7533c-eb9e-11e6-967d-89a68dc61d93.png)

This is a really cool extension btw, thanks for starting it!

**Comment by dmed256** [comment]
Looks like it's the theme.css from readthedocs/sphynx messing the styling
After removing the theme.css parts:

![tabs2](https://cloud.githubusercontent.com/assets/1812355/22628777/e0b892e2-eb9f-11e6-9507-9047e748d082.png)

It looks a lot better but I don't see the tab borders or content box (not sure if that's due to the query-ui css parts)

**Comment by dmed256** [comment]
Changing

```html
<link rel="stylesheet" href="_static/jquery-ui.min.css" type="text/css">
```

to

```html
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.css" type="text/css">
```

seemed to fix it:

![css](https://cloud.githubusercontent.com/assets/1812355/22628889/9a878a88-eba1-11e6-94ff-c2bd3c07c918.png)

I think the screenshot in the second comment looks better than this one since it looks more like readthedocs
Here's a screenshot of what I was aiming to have before I ran into your project from [Spark](http://spark.apache.org/docs/latest/quick-start.html)

![spark](https://cloud.githubusercontent.com/assets/1812355/22628940/82d61700-eba2-11e6-9477-1c86cbf1f1c3.png)

If that looks good to you too, I'll make some updates and send a pull request for it

**Comment by dmed256** [comment]
I rewrote it to use semantic-ui rather than jquery-ui, but heavily based it on your work

![example](https://cloud.githubusercontent.com/assets/1812355/22630827/3a5285bc-ebc6-11e6-93c9-faf7960d3dde.png)

If you want to take a look at it, here's the [link to the code](https://github.com/libocca/occa-docs/blob/master/source/_extensions/tabs.py#L1)

and a temporary example of it running [here](http://occa.readthedocs.io/en/latest/occa/device.html)

Thanks for making your code public, it was super useful to learn from!

**Comment by djungelorm** [comment]
Thanks for the info!

Is there any benefit semantic-ui over jquery? Not come across that one before.

**Comment by dmed256** [comment]
Semantic UI is kinda like Bootstrap, it provides nice looking things like buttons/inputs/menus (probably overkill for this). The only reason I switched to it was because I thought it looked good =P

I added more directives

```
.. group-tab
.. code-tab
```

The original `tab` has a unique id using your `env.new_serialno` method

The `group-tab` and `code-tab` use the tab label as the tab id. If someone clicks `C++`, all `tabs` that have a `tab` named `C++` will get activated

`code-tab` uses CSS to reduce the padding inside the tab content region and hides the border (since the code is the only thing there, why show the box inside the other box?)

Here is a more detailed use of them:

http://occa.readthedocs.io/en/latest/user/getting_started.html

OH! another thing
I didn't like that when I clicked one `group-tab`, contents for that group expand, you lose track of where you were in the page. With some JS-ery, if you click a `group-tab`, you automatically stay where you were (makes more sense if you click a tab in the page I linked)

Since you did the heavy lifting of starting this project, finding how custom directives work, and setup the sphinx_tabs pip package, is this something we can merge with this project? (If not, totally ok ;))
